compiler/setup.py

```diff
@@ -456,11 +456,17 @@ def find_git_submodule_revision(submodule_path):
             # TODO: We have renamed to iree-compile on 2022-03-18. Remove
             # this alias once no longer needed.
             "ireec = iree.compiler.tools.scripts.ireec.__main__:main",
```

Comment:
Unrelated to this PR, but I think we can remove `ireec` (in favor of just `iree-compile`) now.

Reply:
Yeah, was noticing that but didn't want to pollute this patch.